lib/lambda/planning.js

```diff
@@ -595,6 +598,18 @@ const planning = {
     Object.assign(script, {
       _genesis: script._genesis === undefined ? timeNow : script._genesis,
     }),
+
+  removeInvalidPhases: (script) => {
+    const updatedScript = JSON.parse(JSON.stringify(script))
+
+    // Remove phases that have explicitly set a duration to 0
+    const newPhases = updatedScript.config.phases.filter(phase => !('duration' in phase) || phase.duration > 0)
+    const phasesWithDuration = updatedScript.config.phases.filter(phase => 'duration' in phase)
+
+    // Update the phases in the script. If there are no phases with a duration (e.g. only `pause` phases), then remove everything
+    updatedScript.config.phases = phasesWithDuration.length > 0 ? newPhases : []
```

Comment:
nit: I don't know if there is much efficiency to gain but calculating newPhases doesn't need to be done if this test passes

Reply:
Mostly irrelevant now that I've tracked down the actual issue and implemented a more appropriate fix